src/OmniSharp.Host/MSBuild/Discovery/MSBuildLocator.cs

```diff
@@ -34,33 +34,15 @@ protected override void DisposeCore(bool disposing)
 
         public static MSBuildLocator CreateDefault(ILoggerFactory loggerFactory, IAssemblyLoader assemblyLoader, IConfiguration msbuildConfiguration)
         {
-            var useBundledOnly = msbuildConfiguration.GetValue<bool>("UseBundledOnly");
```

Comment:
we could still look at this value and if we detect it to be `true`, we print a warning into the log saying that this is not supported anymore?

the way the configuration works, it does not even need to be part of the typed model anymore for us to be able to extract it manually here (like it is already done)

Reply:
Good idea. Updated with a warning.